test/fixtures/themes/001-deprecations/v1/invalid/post.hbs

```diff
@@ -38,3 +38,5 @@
         {{cover}}
     {{/if}}
 {{/author}}
+
+{{author}}
```

Comment:
Not sure why the tests didn't complain - @AileenCGN Could you pls double check? Thanks :)

Reply:
That's because this fixture is only being tested for v1. Everything good!